### PR TITLE
Support input devices

### PR DESCRIPTION
**Pull request by futuretea**
Signed-off-by: futuretea <Hang.Yu@suse.com>

**Related issues**
https://github.com/harvester/harvester/issues/3119

**Depend PR**
https://github.com/harvester/harvester/pull/3247

### CODE REVIEW

**Review by FrankYang0529 — APPROVED** [review]
Test the branch with rebase master. It works fine. LGTM.

File: internal/provider/virtualmachine/schema_virtualmachine.go [hunk 1]
```diff
@@ -93,6 +93,14 @@ please use %s instead of this deprecated field:
 				Schema: resourceNetworkInterfaceSchema(),
 			},
 		},
+		constants.FieldVirtualMachineInput: {
+			Type:     schema.TypeList,
+			Required: true,
```
Comment (guangbochen) [inline internal/provider/virtualmachine/schema_virtualmachine.go:98]: should it be an optional filed?

Comment (futuretea) [reply internal/provider/virtualmachine/schema_virtualmachine.go]: updated

**Comment by futuretea** [comment]
convert to Draft util PR https://github.com/harvester/harvester/pull/3247 merged

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**